Mathlib/Topology/ProperMap.lean

```diff
@@ -290,8 +299,6 @@ lemma isProperMap_restr_of_proper_of_closed {U : Set X} (hf : IsProperMap f) (hU
 lemma IsProperMap.isClosed_range (hf : IsProperMap f) : IsClosed (range f) :=
   hf.isClosedMap.isClosed_range
 
-@[deprecated (since := "2024-05-08")] alias IsProperMap.closed_range := IsProperMap.isClosed_range
-
```

Comment:
Any particular reason you are deleting this alias? In general, please don't remove aliases in a PR called `feat: Something` unless the alias causes you troubles for some reason

Reply:
This was not intended, I copied the file that I had in the other PR and did not see that this line had disappeared.